wye/templates/workshops/workshop_detail.html

```diff
@@ -51,7 +51,11 @@ <h1 class="no-space">Workshop Details</h1>
                     <li class="list-group-item">
                         <div class="row">
                             <div class="col-xs-4"><strong>Instructors</strong></div>
```

Comment:
@curioswati We need to add github or linkedin link as per tutor provided info profile page 
similar to deactivate account link .